Lib/asyncio/coroutines.py

```diff
@@ -107,6 +108,9 @@ def coroutine(func):
     If the coroutine is not yielded from before it is destroyed,
     an error message is logged.
     """
+    warnings.warn('"@coroutine" decorator is deprecated, use "async def" instead',
```

Comment:
```suggestion
    warnings.warn('"@coroutine" decorator is deprecated since Python 3.8 , use "async def" instead',
```

Reply:
Thanks!
The correction is very valuable.

Reply:
Thanks for cleaning up things, and working on that. It's not that much of a correction than a supplement and a small pet-peeve of mine. I find it sometime difficult to know when things were changed and how pressing the updating needs be, so a small nudge could help downstream developers to update their code.

It also make it a tiny bit more probable that this will be removed in a couple of years.